src/app/ui/editor/standby_state.cpp

```diff
@@ -299,11 +301,8 @@ bool StandbyState::onMouseDown(Editor* editor, MouseMessage* msg)
         int x, y, opacity;
         Image* image = site.image(&x, &y, &opacity);
         if (layer && image) {
-          if (!layer->isEditableHierarchy()) {
-            StatusBar::instance()->showTip(
-              1000, fmt::format("Layer '{}' is locked", layer->name()));
+          if (layer_is_locked(editor))
```

Comment:
Maybe as `transformSelection()` checks for `layer_is_locked()` there is no need to do this here (same for the other cases where `layer_is_locked` is called just before `transformSelection`)